daprdocs/content/en/developing-applications/building-blocks/workflow/howto-author-workflow.md

```diff
@@ -241,4 +376,6 @@ Now that you've authored a workflow, learn how to manage it.
 ## Related links
 - [Workflow overview]({{< ref workflow-overview.md >}})
 - [Workflow API reference]({{< ref workflow_api.md >}})
-- [Try out the .NET example](https://github.com/dapr/dotnet-sdk/tree/master/examples/Workflow)
+- Try out the full SDK examples:
+  - [.NET example](https://github.com/dapr/dotnet-sdk/tree/master/examples/Workflow)
+  - [Python example](todo)
```

Comment:
Add the link here: https://github.com/dapr/python-sdk/tree/master/examples/demo_workflow